incubator/kube-aws-ingress-controller/templates/daemonset.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: {{ template "kube-aws-ingress-controller.name" . }}-skipper-ingress
```

Comment:
Resources names should be based on the fullname template. I'd remove the suffix. The same goes for the deployment below.

`name: {{ template "kube-aws-ingress-controller.fullname" . }}`